tests/openebs-pool-container-failure_test.go

```diff
@@ -110,112 +113,76 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
+			By("Fetching engine file for the experiment")
+			err = exec.Command("wget", "-O", "pool-container-failure-ce.yaml", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-pool-container-failure/engine.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to fetch engine file")
+			//Modify chaos engine spec
+			//Modify Namespace,Name,AppNs,AppLabel of the engine
+			err = exec.Command("sed", "-i", `s/namespace: default/namespace: litmus/g; s/name: target-chaos/name: engine/g; s/appns: 'default'/appns: 'litmus'/g; s/applabel: 'app=nginx'/applabel: 'name=percona'/g`, "pool-container-failure-ce.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to change the fields of the engine")
+			//Modify APP_PVC
+			err = exec.Command("sed", "-i", `/name: APP_PVC/{n;s/.*/              value: "percona-vol1-claim"/}`, "pool-container-failure-ce.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to Modify APP PVC name in engine spec")
+
+			//Creating ChaosEngine
 			By("Creating ChaosEngine")
-			chaosEngine := &v1alpha1.ChaosEngine{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "engine",
-					Namespace: "litmus",
-				},
-				Spec: v1alpha1.ChaosEngineSpec{
-					AnnotationCheck:  "false",
-					EngineState:      "active",
-					AuxiliaryAppInfo: "",
-					Appinfo: v1alpha1.ApplicationParams{
-						Appns:    "litmus",
-						Applabel: "name=percona",
-						AppKind:  "deployment",
-					},
-					ChaosServiceAccount: "pool-container-failure-sa",
-					Components: v1alpha1.ComponentParams{
-						Runner: v1alpha1.RunnerInfo{
-							Image: "litmuschaos/chaos-runner:ci",
-							Type:  "go",
-						},
-					},
-					Monitoring:       false,
-					JobCleanUpPolicy: "delete",
-					Experiments: []v1alpha1.ExperimentList{
-						{
-							Name: "openebs-pool-container-failure",
-							Spec: v1alpha1.ExperimentAttributes{
-								Components: v1alpha1.ExperimentComponents{
-									ENV: []v1alpha1.ExperimentENV{
-										{
-											Name:  "APP_PVC",
-											Value: "percona-vol1-claim",
-										},
-									},
-								},
-							},
-						},
-					},
-				},
-			}
-
-			_, err = clientSet.ChaosEngines("litmus").Create(chaosEngine)
+			err = exec.Command("kubectl", "apply", "-f", "pool-container-failure-ce.yaml", "-n", chaosNamespace).Run()
+			Expect(err).To(BeNil(), "Fail to create chaos engine")
 			if err != nil {
 				fmt.Println(err)
 			}
-			time.Sleep(5 * time.Second)
 
-			fmt.Println("Chaosengine created successfully...")
+			fmt.Println("ChaosEngine created successfully...")
+			time.Sleep(2 * time.Second)
 
 			//Fetching engine-nginx-runner pod
-			runner, err := client.CoreV1().Pods("litmus").Get("engine-runner", metav1.GetOptions{})
+			runner, err := client.CoreV1().Pods(chaosNamespace).Get("engine-runner", metav1.GetOptions{})
 			fmt.Printf("name : %v \n", runner.Name)
 			for i := 0; i < 30; i++ {
 				if string(runner.Status.Phase) != "Succeeded" {
 					time.Sleep(10 * time.Second)
```

Comment:
take the total chaos duration + graceperiod (120s)

Reply:
derive from engine, if empty use experiment

Reply:
done